core/controllers/learner_dashboard_test.py

```diff
@@ -284,7 +284,8 @@ def test_learner_dashboard_page(self):
 
         response = self.get_html_response(feconf.LEARNER_DASHBOARD_URL)
         self.assertIn(
-            '<title itemprop="name">Learner Dashboard - Oppia</title>',
+            '@require(\'../../base-components/header.template.html\', '
+            '{"title": "Learner Dashboard - Oppia"})',
```

Comment:
Here as well as every other place that it is required

Reply:
Updated.